src/System.Net.ServicePoint/ref/System.Net.ServicePoint.cs

```diff
@@ -56,6 +56,7 @@ public enum SecurityProtocolType
     {
         SystemDefault = 0,
 #pragma warning disable CS0618
+        [Obsolete("This value has been deprecated.  It is no longer supported. http://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
nit: The current trend is to have only one space after a period. "deprecated.  It is" -> "deprecated. It is"
ref: http://www.huffingtonpost.com/2015/05/29/two-spaces-after-period-debate_n_7455660.html